tools/openblas_support.py

```diff
@@ -120,7 +121,8 @@ def download_openblas(target, plat, ilp64):
 
     if not suffix:
         return None
-    filename = f'{BASEURL}/openblas{fnsuffix}-{OPENBLAS_LONG}-{suffix}'
+    BASEURL = f'{BASE_LOC}/{openblas_version}/download'
```

Comment:
bear in mind (for the future) that there is cross-pollination between numpy and scipy for this script, we probably don't want to diverge to much from each other, because it makes it harder to take advantage of changes that each one has.

Reply:
That's a good point. If NumPy starts using nightly OpenBLAS in the similar NumPy CI job, the same changes will need to be made there. Do you think it would be workable to refactor the script to be an external dependency of both NumPy and SciPy to avoid having it be maintained in two separate projects?